### PR TITLE
Web - V2

### PR DESCRIPTION
**Pull request by mikejahn**
V2 for Web based on feedback from V1.
Decided to use a simple map to store the usernames and then write/read from file.
Also display the counts in the webpage.

### CODE REVIEW

**Comment by mikejahn** [comment]
@spectreman @wmbest2

File: exercise-003-web/exercise/server.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+		if error != nil {
+			err = error
+		}
+		userNames[line[0]] = count
+	}
+
+	return userNames, err
+}
+
+func updateWordAcount(s string, words map[string]int) map[string]int {
```
Comment (wmbest2) [inline exercise-003-web/exercise/server.go]: Should this be updateWordCount?

File: exercise-003-web/exercise/server.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		}
+	}
+	w.Flush()
+	return err
+}
+
+func readUsernameFile(filename string) (map[string]int, error) {
+	userNames = make(map[string]int)
+	csvFile, _ := os.Open("names.csv")
+	reader := csv.NewReader(bufio.NewReader(csvFile))
```
Comment (wmbest2) [inline exercise-003-web/exercise/server.go]: What happens if you read the files while the other handler is writing it?

File: exercise-003-web/exercise/server.go (hunk 1 above)
Comment (mikejahn) [reply exercise-003-web/exercise/server.go]: Typo! Will fix.

File: exercise-003-web/exercise/server.go (hunk 2 above)
Comment (mikejahn) [reply exercise-003-web/exercise/server.go]: Good point! Would you recommend a Lock here? https://golang.org/pkg/sync/#Mutex.Lock

Comment (wmbest2) [reply exercise-003-web/exercise/server.go]: I believe a https://golang.org/pkg/sync/#RWMutex would be best suited for this kind of thing. @spectreman Might know better

**Pull request closed without merging**